bumble/gatt.py

```diff
@@ -219,9 +227,10 @@ def __init__(self, uuid, characteristics: list[Characteristic], primary=True):
             uuid.to_pdu_bytes(),
         )
         self.uuid = uuid
-        # self.included_services = []
+        self.included_services = included_services
```

Comment:
Should make a copy of the list here, like for `characteristics`, so that if the caller modifies  the list later on, this won't affect the local copy.

Reply:
Done